server/integration/feature-pack/src/main/resources/modules/system/layers/base/org/hibernate/search/engine/main/module.xml

```diff
@@ -8,6 +8,7 @@
         <module name="javax.transaction.api" />
         <module name="org.hibernate.commons-annotations" />
         <module name="org.apache.lucene" export="true" />
+        <module name="org.apache.lucene.internal" export="true" />
```

Comment:
Also, I suppose this change would need to go to the modules we distribute as well https://github.com/infinispan/infinispan/tree/master/wildfly-modules
In that case, we simply overlay whatever the hibernate search module zip gives us, probably we should tell them to export the missing classes...